app/react-components/components/DownloadVisualizationButton.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import DropDownButton from './DropDownButton';
+import downloadSvg from '../utils/download-svg';
+import saveFile from '../utils/filesaver';
+import toTsvString, {mapArrayToTsvString} from '../utils/toTsvString';
+
+function getSelector(el) {
+  return typeof el === 'string' ? document.querySelector(el) : el;
+}
+const styles = {
+  hidden: {
+    width: 0,
+    height: 0,
+    overflow: 'hidden',
+  },
+};
+
+const icon = <i className={"fa fa-download"}/>;
```

Comment:
So no react-icons? is there an alternative, or should I just leave the className?